charter.adoc

```diff
@@ -15,20 +15,20 @@ The Toolchains SIG develops the strategy, identifies gaps, and prioritizes tasks
 
 The SIG's main goals are:
 
-- Push RISC-V into open-source software
-- Pull developers to the RISC-V ecosystem
+- Push RISC-V into open-source toolchain projects
+- Pull developers to the RISC-V toolchain ecosystem
 - Advice on ISA extensions
 
 To achieve these goals, the following steps are taken:
 
-- Advice on ISA extensions and ensure that all future ISA extensions are driven by demonstrable/quantifiable benefits and possibly support them in development tools. Enable the SW HC to drive the ICs to develop extensions necessary to support the strategy.
+- Promote RISC-V as an ISA to be used for research into novel compilation techniques.
+- Track the support for RISC-V across development tools and identify gaps.
 - Identify and work with the key open-source projects for the RISC-V software ecosystem: compilers, debuggers, performance analysis tools, libraries implementing performance primitives, and memory management libraries.
-- Liaise between open-source compiler developers and RISC-V task groups defining ISA, profiling, and debug extensions.
+- Liaise between open-source compiler developers and RISC-V groups defining new extensions.
+- Streamline RISC-V-specific ABIs (e.g. C calling convention), APIs (e.g. intrinsics, attributes), ASM language, and other interfaces (e.g. command line flags) in the RISC-V toolchain ecosystem.
```

Comment:
We could refer to toolchain conventions specs here:
and other interfaces (toolchain conventions, e.g., command line flags)